ethereumetl/domain/trace.py

```diff
@@ -23,6 +23,8 @@
 
 class EthTrace(object):
     def __init__(self):
+        self.before_evm_transfers = None
```

Comment:
Add a comment that its applicable for arbitrum chains. Add details to it please.